modules/network-observability-viewing-alerts.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-alert-dashboard_{context}"]
+= Viewing health information
+
+You can access metrics about the health of the Network Observability Operator from the *Dashboards* page in the web console. A health alert banner that directs you to the dashboard might sometimes appear on the *Network Traffic* and *Home* pages. Alerts are generated in the following cases:
+
+* If the `flowlogs-pipeline` workload is dropping flows because of Loki errors, such as if the Loki ingestion rate limit has been reached
+* If no flows are ingested for a certain amount of time
+
+.Prerequisites
+
+* You have the Network Observability Operator installed.
+* You have access to the cluster as a user with the `cluster-admin` role or with view permissions for all projects.
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Observe* → *Dashborad*.
+. From the *Dashboards* dropdown, select *Netobserv/Health*. 
+. Metrics about the health of the Operator is displayed on the page.  
+
+== Disable health alerts
+Health alerts are automatically enabled. You can opt out of health alerting by editing the YAML:
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+  Processor:
+    Metrics:
+      DisableAlerts: NetObservLokiError NetObservNoFlows <1>
```

Comment:
If you wanted to specify both alerts, would it be like this? Or is a delimiter needed? @OlivierCazade

Reply:
- "processor", "metrics" and "disableAlerts" need low case first letter
- the list of alerts to disable must be coma-separated, and enclosed in brackets.

so it's:

```yaml

apiVersion: flows.netobserv.io/v1alpha1
kind: FlowCollector
metadata:
  name: cluster
spec:
  processor:
    metrics:
      disableAlerts: [NetObservLokiError, NetObservNoFlows] <1>
```

Reply:
This can also be done like this:
```      
    disableAlerts:
        - NetObservLokiError
        - NetObservNoFlows
```

This is personnal choice, but I tend to find this more readable.

Reply:
I'll go with the comma-separator just to keep the options on one line.